### ISSUE TITLE
Cannot find module '../lib/load-env-config'.

### ISSUE THREAD

**Issue by comxd** [zeit/next.js#11396]
I use a similar structure of [this example](https://github.com/zeit/next.js/tree/canary/examples/with-firebase-hosting-and-typescript), during compilation of functions:
```
tsc --project src/functions
```

I got this error:
```
node_modules/next/types/index.d.ts:8:21 - error TS2307: Cannot find module '../lib/load-env-config'.
```

## System information

- OS: (K)Ubuntu 19.10
- Version of Node: 10.19.0
- Version of Next.js: 9.3.2

**Comment by timneutkens** [comment]
Please provide a full reproduction as asked for in the issue template

**Comment by pcfields** [comment]
I got the same error after using the the **with-firebase-hosting-and-typescript** example project and running `npm run deploy`

**Comment by comxd** [comment]
Fixing the path in `node_modules/next/types/index.d.ts` working good:
`import { Env } from '../dist/lib/load-env-config';`

I can provide more details later.

**Comment by combstraight** [comment]
Getting the same error.    

> Fixing the path in `node_modules/next/types/index.d.ts` working good:
> `import { Env } from '../dist/lib/load-env-config';`
> 
> I can provide more details later.

This fixed it as well.

**Comment by arusakov** [comment]
Use [patch-package](https://www.npmjs.com/package/patch-package) for repeatable fix after each `yarn install` / `npm ci`.

**Comment by Timer** [comment]
Fixed in 9.3.3

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
